rust/src/fvm/machine.rs

```diff
@@ -11,7 +11,7 @@ use fvm4_shared::address::Address;
 use fvm4_shared::state::ActorState;
 use fvm4_shared::{ActorID, MethodNum};
 use fvm_ipld_encoding::ipld_block::IpldBlock;
-use fvm_ipld_encoding::tuple::{Deserialize_tuple, Serialize_tuple};
+use fvm_ipld_encoding::tuple::*;
```

Comment:
We usually avoid this, but I know you know that and have reasoned about it already

Reply:
Basically, I need to either import the `serde_tuple` re-export or include the crate directly. But if I include the crate directly, I need to make sure that all the versions agree.

The mistake was re-exporting anything from `fvm_ipld_encoding`, but changing that is a larger breaking change.

Reply:
This became an issue because `serde_tuple` released a v1.0 and we're _not_ updating to that because (a) it doesn't introduce any important new features/fixes and (b) making breaking changes to `fvm_ipld_encoding` is really painful.